architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,143 @@ endif::[]
+Image Lookup:		local=false
+Unique Images:		2
+Tags:			2
+
+34
+  tagged from centos/python-34-centos7
+
+  * centos/python-34-centos7@sha256:28178e2352d31f240de1af1370be855db33ae9782de737bb005247d8791a54d0
+      14 seconds ago
+
+35
+  tagged from centos/python-35-centos7
+
+  * centos/python-35-centos7@sha256:2efb79ca3ac9c9145a63675fb0c09220ab3b8d4005d35e0644417ee552548b10
+      7 seconds ago
+----
+
+Using image streams has several significant benefits:
+
+* You can tag, rollback a tag, and quickly deal with images, without having to re-push using the command line.
+
+* You can trigger Builds and Deployments when a new image is pushed to the registry. Also,
+{product-title} also has generic triggers for other resources (such as Kubernetes objects).
+
+* You can xref:../../dev_guide/managing_images.adoc#importing-tag-and-image-metadata[mark a tag for periodic re-import].
+If the source image has changed, that change is picked up and reflected in the image stream and can trigger the Build and/or Deployment flow, depending upon the Build or Deployment configuration.
+
+* You can share images using fine-grained access control and quickly distribute images across your teams.
+
+* If the source image changes, the image stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+* You can configure security around who can view and use the images through permissions on the image stream objects. 
+
+* Users that lack permission to read or list images on the cluster level can still retrieve the images tagged in a project using image streams.
+
+
+For a curated set of image streams, see the
 link:https://github.com/openshift/library[OpenShift Image Streams and Templates
 library].
 
-For example, if a deployment is using a certain image and a new version of that
-image is created, a deployment could be automatically performed.
+
+// https://access.redhat.com/solutions/3016781 Caveats around referencing latest tags
+When using Docker images it is important to understand what image your image stream tag is pointing to and be aware if the image that the image stream tag is pointing to can change.
+For example, if pointing to the `latest` image stream tag and the version of the `latest` image changes, the new version might not be backwards-compatible with your applications.
+Using an image stream with a version-specific image stream tag will prevent problems using `latest` by providing a stable pointer until the next import.
+
+=== Important terms
+
+Docker repository::
+A central location for uploading and downloading images.
+For example, the OpenShift Jenkins images are in a Docker registry:
++
+----
+docker.io/openshift/jenkins-2-centos7
+----
+
+Docker image::
+A reference to an image for a given external registry. It uses the standard Docker pull specification for its name, for example:
+
+----
+openshift/jenkins-2-centos7
```

Comment:
a docker image is the physical collection of content.  it's what you get when you "pull" a particular docker image tag.

to the extent that it has a name at all, it would be the sha id, not what is shown here.  I would probably skip showing the name entirely.